src/advancedKubectl/content.md

```diff
@@ -0,0 +1,1108 @@
+If you work with Kubernetes, then kubectl is probably one of your most-used tools. Whenever you spend a lot of time working with a specific tool, it is worth to get to know it very well and learn how to use it efficiently.
+
+This article contains a series of tips and tricks to help you boost your kubectl productivity. At the same time, it aims at deepening your understanding of how various aspects of Kubernetes work.
+
+The goal of this article is not only to make your daily work with Kubernetes more efficient but also more enjoyable!
+
+## Contents
+
+- [**Introduction: what is kubectl?**](#introduction-what-is-kubectl-)
+- [**1. Save typing with command completion**](#1-save-typing-with-command-completion)
+- [**2. Quickly look up resource specifications**](#2-quickly-look-up-resource-specifications)
+- [**3. Use the custom columns output format**](#3-use-the-custom-columns-output-format)
+- [**4. Switch between clusters and namespaces with ease**](#4-switch-between-clusters-and-namespaces-with-ease)
+- [**5. Save typing with auto-generated aliases**](#5-save-typing-with-auto-generated-aliases)
+- [**6. Extend kubectl with plugins**](#6-extend-kubectl-with-plugins)
+
+## Introduction: what is kubectl?
+
+Before learning how to use kubectl more efficiently, you should have a basic understanding of what it is and how it works.
+
+From a user's point of view, kubectl your cockpit to control Kubernetes. It allows you to perform every possible Kubernetes operation.
```

Comment:
Missing verb. kubectl **is** your cockpit to control Kubernetes

Reply:
Thanks